lms/djangoapps/branding/tests/test_views.py

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+"""
+Tests of branding api views.
+"""
+
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+import json
+from branding.models import BrandingApiConfig
+
+
+class TestFooter(TestCase):
+    """ Test for getting the footer data as json
+    """
+
+    def test_footer(self):
+        """ Test the footer json
+        """
```

Comment:
Since this docstring and the one preceeding it are a single line, put the closing quotes on the same line. For example:
```python
"""Test retrieval of footer JSON.
```
Also, please capitalize JSON where appropriate.